src/handlers/platformProviderResponsesHandlers.ts

```diff
@@ -0,0 +1,124 @@
+import { ajv } from '../ajv';
+import { db } from '../database';
+import { isTinyPgErrorWithQueryContext } from '../types';
+import {
+  DatabaseError,
+  InputValidationError,
+} from '../errors';
+import type {
+  Request,
+  Response,
+  NextFunction,
+} from 'express';
+import type { JSONSchemaType } from 'ajv';
+import type { PlatformProviderResponse } from '../types';
+
+interface GetPlatformProviderResponseByExternalIdParams {
+  externalId: string;
+}
+const getPlatformProviderResponsesByExternalIdParamsSchema
+: JSONSchemaType<GetPlatformProviderResponseByExternalIdParams> = {
+  type: 'object',
+  properties: {
+    externalId: {
+      type: 'string',
+      pattern: '^\\d{9}$',
+    },
+  },
+  required: [
+    'externalId',
+  ],
+};
+const isGetPlatformProviderResponsesByExternalIdParams = ajv.compile(
+  getPlatformProviderResponsesByExternalIdParamsSchema,
+);
+const getPlatformProviderResponsesByExternalId = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void => {
+  if (!isGetPlatformProviderResponsesByExternalIdParams(req.query)) {
+    next(new InputValidationError(
+      'Invalid request parameters.',
+      isGetPlatformProviderResponsesByExternalIdParams.errors ?? [],
+    ));
+    return;
+  }
+  db.sql<PlatformProviderResponse>(
+    'platformProviderResponses.selectByExternalId',
+    { externalId: req.query.externalId },
+  ).then((platformProviderResponsesQueryResult) => {
+    const { rows: platformProviderResponses } = platformProviderResponsesQueryResult;
+    res.status(200)
+      .contentType('application/json')
+      .send(platformProviderResponses);
+  }).catch((error: unknown) => {
+    if (isTinyPgErrorWithQueryContext(error)) {
+      next(new DatabaseError(
```

Comment:
*sigh* idk -- do we need to test these, should I give up on that dream.  It made sense for the first endpoint or two but now It's just so noisy and so much duplicated test code.  (See #228 which I think speaks to this issue indirectly).  Happy to add it, but also happy to just move on with our lives.